pyqt-apps/siriushla/si_di_equalize_bpms/main.py

```diff
@@ -394,17 +436,19 @@ def _save_orbit(self):
             _log.error('Must Acquire and process data first.')
             return
         data = dict()
-        data['x'] = self._orbits['x'] + dorbx
-        data['y'] = self._orbits['y'] + dorby
+        data['x'] = self._orbits['x'] - dorbx
+        data['y'] = self._orbits['y'] - dorby
```

Comment:
Why is this being changed here?

https://github.com/lnls-sirius/dev-packages/blob/369cb8a7116839456f57560a444368d6c89ed8b3/siriuspy/siriuspy/devices/bpm_eq.py#L431-L432

has

```
        dorbx = orbx_new - orbx_init
        dorby = orby_new - orby_init
```

so `new = init + dorb`, not `init - dorb`.

Reply:
Because I tried the old way and it didn't work. Then I tried the new way and it worked.
I didn't have time to analyze this yet, though...
Could it be that the way I'm calculating the the orbit has a wrong minus sign in relation to the way the BPMs do?
I need to verify this with real data from the machine...

Reply:
I took a look at how the orbit is calculated in your code, and it seemed correct.

Reply:
I feel like it'd be best to fix this at the dev-packages level either way, no. And keep the `+ dorb` here.

Will try to run that block.

Reply:
yes, now that we've found out the reason, I'll change there. But I don't know where this signal comes from, given that you said the way I do it is similar to what the BPM does...

Reply:
I'll fix the PR in dev-packages.

Reply:
@fernandohds564 

Oh, I think it's the diff!

```
In [37]: np.diff(ants[::2], axis=0).squeeze()
Out[37]:
array([[-359312,   76203, -256106, ...,  296167,  141899,   46304],
       [-348274,   74021, -256578, ...,  278749,  141583,   41923],
       [-356545,   74941, -256357, ...,  284297,  144141,   42356],
       ...,
       [-346711,   71710, -257158, ...,  279000,  137253,   41850],
       [-355885,   72235, -257176, ...,  283098,  140501,   40869],
       [-350946,   74310, -256357, ...,  290623,  137005,   46650]],
      dtype=int32)

In [38]: ants[0]-ants[2]
Out[38]:
array([[ 359312,  -76203,  256106, ..., -296167, -141899,  -46304],
       [ 348274,  -74021,  256578, ..., -278749, -141583,  -41923],
       [ 356545,  -74941,  256357, ..., -284297, -144141,  -42356],
       ...,
       [ 346711,  -71710,  257158, ..., -279000, -137253,  -41850],
       [ 355885,  -72235,  257176, ..., -283098, -140501,  -40869],
       [ 350946,  -74310,  256357, ..., -290623, -137005,  -46650]],
      dtype=int32)
```

We want `a - c` and `b - d`, but diff will do `c - a` and `d - b`